### DISCUSSION TITLE
My profile is not visible by others. However, I have full access to my profile

### DISCUSSION THREAD

**Discussion by taufiq-nlp** [community/community#55609] [New to GitHub]
https://github.com/taufiq-nlp

Here is my github link. No one at my organization can find my profile page. Additionally, they can not see my pull requests.

**Comment by queenofcorgis** [comment]
HI there 👋🏼  @taufiq-nlp

Confirming when I click on that link I can see your profile as well as when I do a search. Are you still experiencing this?

**Reply by taufiq-nlp** [reply]
No. It was fixed.

**Reply by ADITYAbasude** [reply]
How, I also have a same issue

**Reply by suresha97** [reply]
Hi, I filled out the reinstatement request a few days ago as you suggested. Do you know how long it will take ?

**Reply by Pasindu-Jayasundara** [reply]
how did you fix it

**Reply by d4nyl01** [reply]
i can't fix it too

**Reply by A-Tamerr** [reply]
> HI there 👋🏼 @taufiq-nlp
> 
> Confirming when I click on that link I can see your profile as well as when I do a search. Are you still experiencing this?

I'm experiencing the exact same thing with my profile

**Reply by kafishabbir** [reply]
they did this to my good account https://github.com/kafiulshabbir they will not restore it, for 2 months there was no response, i swear if i will ever be in the united states, i will find them

**Reply by hillarymelina** [reply]
I am having the same issue, what can I do to fix this ? T.T mine is https://github.com/hillarymelina

**Reply by Pe225** [reply]
I have problem:
https://github.com/Pe225/

**Reply by qxdFox** [reply]
seems like commenting on this fixed it for some people? I'll try my luck https://github.com/qxdFox

**Reply by taimurey** [reply]
having the same problem

**Reply by Tsundgerhen** [reply]
i have same problem to. https://github.com/tsundgerhen

**Reply by PedroVSX** [reply]
I'm having the same problem too
https://github.com/PedroVSX

**Reply by Joshuadmello04** [reply]
im facing the same issue https://github.com/Joshuadmello04

**Reply by ThinhGM09** [reply]
https://github.com/cat2d2/io

**Reply by adith2005-20** [reply]
i have the same problem too, someone please help me out https://github.com/adith2005-20

**Reply by crystalcursed** [reply]
@adith2005-20 How did you solve the problem??

**Reply by hillarymelina** [reply]
Mines was fixed almost immediately. I just contacted support and followed instructions.

**Reply by StudiYash** [reply]
> Mines was fixed almost immediately. I just contacted support and followed instructions. 

How did you contact support ?
Any links for that or any process to do for that ?

**Reply by MohammedAlmajhali** [reply]
I'm facing the same issue https://github.com/MohammedAlmajhali

**Reply by Moham3dMahmoud** [reply]
same thing here

**Reply by Chukwuebuka-2003** [reply]
Please I need this to be fixed. My colleagues at work can't see my GitHub profile 


![IMG-20250822-WA0012.jpg](https://github.com/user-attachments/assets/afdecc1e-0aaf-486c-9b5a-b648a51b4851)

**Reply by Salman-Nazeer-477** [reply]
Having the same problem https://github.com/Salman-Nazeer-477

**Reply by JoseRamonRenteroJimenez** [reply]
I am now having the same problem here https://github.com/JoseRamonRenteroJimenez/

**Reply by ANHKIET172004** [reply]
I am now having the same problem https://github.com/ANHKIET172004

**Reply by rotten-ttc** [reply]
I am having the same problem https://github.com/rotten-ttc
I tried contacting support but the verification SMS never arrives

**Reply by umaribnanas** [reply]
https://github.com/umaribnanas

same here! I even finished 2fa and all that stuff, but it still says it's flagged.

 I contacted support and raised a ticket, I hope they solve this for everyone.

**Reply by regartzpris** [reply]
I am having the same problem  https://github.com/regartzpris

**Reply by GuztavoRbr** [reply]
I am having the same problem on  https://github.com/GuztavoRbr

**Reply by danielzikol** [reply]
I am having the same problem!
here is my link: https://github.com/danielzikol
If people google it, it does not appear on the search :/

**Reply by kleZ799** [reply]
yeah man same problem

**Reply by Vincent-P-essy** [reply]
same problem for me

**Comment by thaciogm** [comment]
I am having the same problem. @taufiq-nlp , What did you do to solve it?

**Reply by taufiq-nlp** [reply]
Contacted github support

**Reply by Zile10** [reply]
Github support doesnt get back to me. Its been a month and a half.

**Reply by rooneyrulz** [reply]
@taufiq-nlp  How can I do that? I am having the same problem. I can login to my account but It's not visible to others. I already created Reinstatement Request in github support here https://support.github.com/contact/reinstatement. Is that how you fix? or I should do something else?

**Reply by sefinek-actions** [reply]
@rooneyrulz Have you resolved it already?

**Reply by Anonav0** [reply]
It is visible now.

**Reply by Vincent-P-essy** [reply]
contact github support

**Comment by jamesdev0409** [comment]
I am having the same issue. @taufiq-nlp How did you resolve it?

@thaciogm Did you fix it?

**Reply by taufiq-nlp** [reply]
Contacted github support team

**Comment by akashdip2001** [comment]
Sure, here's a step-by-step guide:

1. **Profile Visibility:**
   - Make sure your profile visibility settings are set to public.
     - Go to your GitHub account Settings.
     - Navigate to the "Account visibility" section.
     - Set it to "Public" if it's not already.

2. **Repository Visibility:**
   - Ensure that the repositories containing your pull requests are either public or accessible by your organization members.
     - Go to your repository Settings.
     - Under "Danger Zone" (at the bottom), check if the repository is private or public. Adjust accordingly.

3. **Pull Request Visibility:**
   - Confirm that your pull requests are not in a private repository or marked as drafts.
     - Check the repository where you've submitted pull requests.
     - Ensure the repository is public or accessible by your organization.
     - Verify that your pull requests are not marked as drafts.

4. **Share Direct Links:**
   - If your colleagues still can't find your profile or pull requests, consider sharing direct links to your profile or pull requests.

## Profile
![Profile Image](https://thumbnails.odycdn.com/optimize/s:200:0/quality:95/plain/https://thumbs.odycdn.com/bf58cf4d7ec52c62f02c52e16f150a5b.webp)
**AKASHDIP MAHAPATRA**
"everything is physics bro"  
E=mc²

|[Subscribe on YouTube](https://www.youtube.com/channel/UCxvmp634YDc41xCWOdvWqoQ?sub_confirmation=1) |
|----  |

**Reply by WebGaidos** [reply]
Hi Akashdip, Your advice looks great, but I don't see an "Account Visibility" section in my github account settings.  Where should I find the "Account Visibility" settings?

**Reply by mxamak** [reply]
neither do I, i got stuck at the second step. "nAvIgAtE tO aCcOuNt vIsIbIlITy"

**Reply by beyond88** [reply]
@akashdip2001 you just generated the answer from ChatGPT and paste it here.

**Comment by theburcu** [comment]
Same thing is happening to me too. I have access to my profile, however I can't check how my profile looks from an incognito window. How did you contact the support team?

**Reply by tariqkhan051** [reply]
@theburcu contact suport by filling out the form: 
https://support.github.com/contact/reinstatement?tags=dotcom-contact-params

**Reply by StudiYash** [reply]
> @theburcu contact suport by filling out the form: 
> https://support.github.com/contact/reinstatement?tags=dotcom-contact-params


Did your problem get fixed by filing out this form? 
I am facing the same right now.

**Reply by wdcodecn** [reply]
I encountered the same problem https://github.com/wdcodecn, have sent a ticket #3300964 for a month but no response .

**Comment by taufiq-nlp** [comment]
> https://github.com/taufiq-nlp
> 
> Here is my github link. No one at my organization can find my profile page. Additionally, they can not see my pull requests.

Already resolved after contacting supoort that day.

**Reply by ADITYAbasude** [reply]
How much days they take to reply you??

**Reply by adem-mehdioui** [reply]
i'm having the same problem, my github account is not visible to others although i can still log in , 
when someone tries to access my github account link , it shows him : 404 not found ,  
which means my account is not visible to others, 
i've contacted github support ,  three days ago, and it is still not working.

how many days will they restore my account ?

**Reply by jjinendra3** [reply]
Hey @ADITYAbasude , did yours get fixed, if yes then how much time did gh took to fix it?

**Reply by ADITYAbasude** [reply]
Yeah it's fixed now, taken something 1-2 months.

**Reply by engageub** [reply]
I have the same problem. Today I reported to github support.

**Reply by sefinek-actions** [reply]
@beyond88 Your account has been restored?

**Reply by beyond88** [reply]
Yes, I did it.

**Comment by NarmadhaKandasamy** [comment]
I am experiencing a similar issue with https://github.com/GauthamSivasubramanian. I can't access this profile in public, and others can't access the PR that I created.

**Reply by ADITYAbasude** [reply]
Contact to the GitHub support they can fix this, i also suffer from this issue.

**Reply by Zile10** [reply]
> Contact to the GitHub support they can fix this, i also suffer from this issue.

They don't get back to me! My ticket has just been sitting there for more than 2 months now, and github support is just ignoring it! What am I supposed to do now?

**Reply by darshit145** [reply]
hi NarnadhaKandasmy Did your issue resolved cause i have similar kind of profile,and if all is well then how did you done it

**Comment by adem-mehdioui** [comment]
Hello,
it's fixed , i've raised a ticket and the github community team has
resolved the issue, thank you

Le jeu. 1 août 2024 à 10:18, aditya basude ***@***.***> a
écrit :

**Reply by engageub** [reply]
Hello,
I have the same problem. I raised the issue and they have not responded yet.
Could you please let me know how long it took for resolving. 

Thank you

**Reply by rooneyrulz** [reply]
@engageub were you able to recover your account?  I am having the same issue with my github account. I created a reinstatement request here https://support.github.com/contact/reinstatement but I did not know it is the right way to recover it. is your account fixed now?

**Reply by engageub** [reply]
Yes, the issue was with multiple users logging in from same IP address. Its resolved after discussing with support team.

**Comment by ghost** [comment]
I am a new user of github. I have 3 accounts on github. But all my accounts are blocked, I don't know why. Tried contacting github support several times.
They did not answer, what should I do?
Can you help me please?

**Comment by MichaelAgarkov** [comment]
Same with my account right now. Fortunately I also have GitLab, Gitea and git.community.i2p, but that's still not normal.

**Comment by Anonav0** [comment]
It is visible now. Please mark the discussion as answered. It is shown on the forum as unanswered.

**Comment by fahimsts** [comment]
https://github.com/fahimsts
My profile is not showing anyone and
when i pull branch it get error

**Comment by RemoconTV** [comment]
Test 1 2 3
https://github.com/RemoconTV

**Comment by iamapuneet** [comment]
https://github.com/iamapuneet

Same with my account right now.
![image](https://github.com/user-attachments/assets/12953cf4-d659-4ab6-9937-3737398115c7)

**Comment by Beijia9574** [comment]
you can try [Avatalk Digital Business Card](https://avatalk.me/?utm_source=github), as it's the first AI-powered [digital business card](https://avatalk.me/?utm_source=github). acts as your 24/7 personal assistant.

**Comment by N0E-cm** [comment]
🌟 Découvrez mon livre de recettes économiques ! 🍽️💰

Vous voulez cuisiner de bons petits plats sans exploser votre budget ? 🛒✨
J’ai créé "Saveurs de Polynésie à petit prix", un livre rempli de 7 jours
de menus économiques et savoureux !

🥥 Ce que vous y trouverez :
✅ Des repas simples et équilibrés, parfaits pour toute la famille
✅ Des ingrédients faciles à trouver et abordables
✅ Des astuces pour économiser sur vos courses
✅ Un guide pratique pour mieux organiser vos repas

💰 Prix : 15 USD (~1640 XPF)
📩 Disponible immédiatement après achat !

📲 Commandez ici 👉https://noemigrove.gumroad.com/

🔁 Partagez cette publication avec vos proches pour les aider à mieux
manger sans se ruiner ! 🙌

Noémie

Le sam. 15 mars 2025 à 8:33 PM, MOHAMED9267 ***@***.***> a
écrit :

**Comment by qxdFox** [comment]
https://github.com/qxdFox/
My Profiles also not showing up at all in google
Other search engines have no issue

**Reply by akashdip2001** [reply]
Try on Chat GPT (internal browser)

**Comment by StudiYash** [comment]
I am facing the same problem right now. Just now I have filled this form :-

https://support.github.com/contact/reinstatement?tags=dotcom-contact-params

Is this the correct form to fill ?
Please someone help me out.

**Reply by wdcodecn** [reply]
https://github.com/wdcodecn

Same with my account right now.

**Comment by abhimangs** [comment]
Hey @GitHubSupport / @queenofcorgis ,

I’m having the exact same issue as others in this thread—my profile ([github.com/abhimangs](https://github.com/abhimangs)) is invisible to everyone else, even though I can access it normally. I already submitted a reinstatement request ([support.github.com/contact/reinstatement](https://support.github.com/contact/reinstatement)) but haven’t gotten a response yet.

Could you help escalate this? It’s blocking my work since my team can’t see my PRs or repos.

**Comment by budi7889** [comment]
Hi, my profile can't showing by others
https://github.com/budi7889

**Comment by evanvolgas** [comment]
I'm having a similar issue in which there is no link to my profile on a public organization's repo of which I am a member

**Reply by StudiYash** [reply]
Hey Evan,
My name is Yash Shukla. I have faced the same issue that you are going through right now.

You can fill the reinstatement form on this [link](https://support.github.com/contact/reinstatement)

After filling this form, the GitHub support team will unlock your account in about 2-3 days maximum. 

I hope your problem solves at the earlist.

**Reply by crystalcursed** [reply]
Hello Yash, it's been 3 months yet my acc has not been reinstated. Do you have any idea how I should proceed ahead?

**Comment by whatapi-xyz** [comment]
i have same problem.

**Comment by Hanzamas** [comment]
I have the same problem

**Comment by dlzvy** [comment]
same problem with me https://github.com/dlzvy

**Comment by votexdev** [comment]
same problem here https://github.com/votexdev

**Comment by rkduongnt5** [comment]
I’m having the same problem with my account: [https://github.com/rk-duongnt5](https://github.com/rk-duongnt5). I’ve already contacted GitHub Support about it. Could anyone share how long it normally takes to get a response from GitHub Support?

**Comment by aqibmunir8** [comment]
i'm en-countering the same issue  it is showing 404 to others! : 
<img width="1920" height="1080" alt="Screenshot (1372)" src="https://github.com/user-attachments/assets/4e2e13f4-f110-4980-b0e4-8f36c0f5f873" />

**Comment by pjdigitalmarketing** [comment]
I am also facing the same issue:

https://github.com/pjdigitalmarketing

Contacted support team, waiting for the response. Check its live or not.

**Reply by pjdigitalmarketing** [reply]
<img width="1919" height="1009" alt="Screenshot 2026-02-09 095213" src="https://github.com/user-attachments/assets/5a8b3d73-c07b-4949-9e36-35cda1226f97" />

**Reply by LuismiLabs** [reply]
Were you able to solve it?

**Reply by pjdigitalmarketing** [reply]
yes, contacted support team, they helped to solve it. Thank You.

**Reply by bhardwajparth51** [reply]
@pjdigitalmarketing how time did they take

**Comment by gigachad80** [comment]
Same here
<img width="1918" height="1032" alt="image" src="https://github.com/user-attachments/assets/9f45fcff-ba34-47a7-8934-987d61cdcd6d" />

**Comment by MohamedAminGrami** [comment]
Contact support and give them your informations and it will be solved

**Reply by umaribnanas** [reply]
Yes did that, inshaallah

**Comment by heyteacher** [comment]
My account in invisible to other people, when I'm logged I see it but I face other issues :

- `issues`, `pull requests`, `tags`, `releases` are invisible but exists

  <img width="1041" height="305" alt="image" src="https://github.com/user-attachments/assets/19a9d3e1-50b3-49ca-ae09-5bf66a9babfb" />

- `workflow` fired on pushing tags doesn't start

- `pull requests` merging doesn't delete `branch` and `issues` remain open.

- `pull request` aren't linked to `issues`
 
3 days ago I contacted Github Support (Reinstatement Request). Next week, I'll migrate all to [codeberg](https://codeberg.org/).

**Reply by heyteacher** [reply]
**UPDATE**: I completed the migration to <https://codeberg.org/>, a public istance of <https://forgejo.org/> a very good software very similar to GitHub (obviously  based on my usage).

The big difference is that <https://codeberg.org/> works as expected , my repos and my account <https://codeberg.org/heyteacher>  are visible. 

A (incomplete) list of `Codeberg` alternatives (public instances of `forgejo`) can by found here: <https://codeberg.org/forgejo-contrib/delightful-forgejo#user-content-public-instances>.

So `GitHub` could leave my account and my projects invisible, doesn't matter.

All my repo in GitHub are now archived except forks because I have to study if is possible to fork e `Github` repo in `Codeberg`, perhaps `mirrors`. 

Furthermore,  I have to study `actions` in `forgejo` in order to publish on `pub.dev` automatically when new remote tag in create. 

I wrote a guide to migrate repos from `Github` to a public instance of `forgejo`: <https://pub.dev/packages/flutter_heyteacher_meta#migrate-from-github-to--forgejo-public-instance>

**Comment by AlexanderSchuetz97** [comment]
Well, it also hit me today...

This is very annoying was looking to get some work done but how can you do that when you cant even fork the repositories to fix bugs.
Let alone anyone seeing your pr's or even issues.

I get that this may be done for brand new acounts who spam issues, but I have been using github in a pretty consistent manner for like almost a decade now.

Edit: I have just realized that this comment is also hidden in a private window so yeah... The only way anyone sees this comment is if github decides to unflag my account. I wonder how many people had their problem not resolved.

**Comment by Infinite-Null** [comment]
Hey @githubsupport / @queenofcorgis,
I am facing same issue with my GitHub https://github.com/Infinite-Null. Can you please fix this ASAP as this is effecting my work!

**Comment by umaribnanas** [comment]
I have fixed it. It seems only 1 account per email/mobile number. I created another one so the new one was flagged so it cannot be seen public. I transferred my repos to the new one and deleted the old one. I contacted support and they said to do that.

Please [contact support](https://support.github.com/) before doing the same. Maybe it might be a different issue for you.

**Comment by karta8931798828** [comment]
I have same problem here https://github.com/karta8931798828

**Comment by mrayhanfadil** [comment]
i have same issue https://github.com/mrayhanfadil

**Comment by Skip-Bug** [comment]
Hi i have same problem to https://github.com/Skip-Bug
<img width="1079" height="550" alt="Screenshot 2026-08-19 125852" src="https://github.com/user-attachments/assets/360ea62f-4bbd-445a-b808-b31b55499499" />
Support is unavailable to me
<img width="1112" height="542" alt="Screenshot 2026-09-03 130221" src="https://github.com/user-attachments/assets/99a10e30-960e-41c7-99c7-c26d35f44ee0" />

**Comment by aleksandarns1991** [comment]
Same here https://github.com/aleksandarns1991

**Reply by kleZ799** [reply]
i can see your profile

**Reply by kleZ799** [reply]
https://github.com/kleZ799 can see mine

**Reply by aleksandarns1991** [reply]
Yes

**Comment by GribLeo** [comment]
Same here  https://github.com/Skip-Bug
<img width="583" height="520" alt="Screenshot 2026-09-03 222712" src="https://github.com/user-attachments/assets/264c66d5-420c-4c57-b5bd-dd7d35b055a9" />

**Reply by kleZ799** [reply]
i raised the ticket , they fixed it

**Reply by GribLeo** [reply]
Hi, thanks, but the page is still invisible.

**Reply by kleZ799** [reply]
give it a day or two it will be fixed automatically
